### DISCUSSION TITLE
Including Python files doctstring

### DISCUSSION THREAD

**Discussion by chaitti** [mkdocs/mkdocs#3452] [Q&A] [answered]
I am using mkdocs to generate for my code. Here to include my docstrings which are very detailed i am using:
::: path.to.my.python.file  in the .md file 

This works well as i get the proper documenation but my source code is also displayed , is there any way i can get ore control over what i want to display and what not in this page?

**Comment by pawamoy** [comment, marked as answer]
Hello, this is a question for mkdocstrings more than MkDocs, and you're looking at the `show_source` option: https://mkdocstrings.github.io/python/usage/configuration/general/#show_source
